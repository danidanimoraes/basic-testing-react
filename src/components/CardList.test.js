import React from 'react';
import { shallow, mount, render } from 'enzyme';
import CardList from './CardList';
import "../../setupTests";


it('should shallow match snapshot CardList', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John',
            username: 'johnjohn',
            email: 'john@gmail.com'
        }
    ]
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot()
});