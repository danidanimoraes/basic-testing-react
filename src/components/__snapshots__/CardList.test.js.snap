// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should shallow match snapshot CardList 1`] = `ShallowWrapper {}`;
